src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -211,6 +212,10 @@ function MoneyRequestConfirmationList({
 }: MoneyRequestConfirmationListProps) {
     const policy = policyReal ?? policyDraft;
     const policyCategories = policyCategoriesReal ?? policyCategoriesDraft;
+    const [mileageRatesDraft] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_DRAFTS}${policyID}`, {
```

Comment:
@dominictb It looks like this caused a regression: https://github.com/Expensify/App/issues/47914